client/app/components/CreateEventModal.jsx

```diff
@@ -0,0 +1,181 @@
+import React, { PropTypes } from 'react';
+import { Button, Modal, FormGroup, ControlLabel, FormControl, Alert } from 'react-bootstrap';
+import DateTimeField from 'react-bootstrap-datetimepicker';
+import moment from 'moment';
+import axios from 'axios';
+import _ from 'lodash';
+
+const { func, bool, array } = PropTypes;
+
+const DATE_FORMAT = 'DD/MM/YYYY HH:mm';
+
+export default class CreateEventModal extends React.Component {
+  static propTypes = {
+    openModal:       func,
+    closeModal:      func,
+    showModal:       bool,
+    conferenceRooms: array
```

Comment:
I'm pretty sure some or all of these props are required so you should mark them as such